addons/toolbars/src/constants.ts

```diff
@@ -1,2 +1,2 @@
-export const ID = 'addon-toolbars' as const;
-export const PARAM = 'toolbars' as const;
+export const ADDON_ID = 'addon-toolbars' as const;
```

Comment:
might as well update to conform with normal usage of this - that it was just called ID threw me off a few times